src/main/java/seedu/address/logic/commands/InviteCommand.java

```diff
@@ -129,7 +130,10 @@ public CommandResult execute(Model model) throws CommandException {
             result = String.format(MESSAGE_RESULT, successMessage, warningMessage);
         }
 
-        return new CommandResult(result);
+        Context newContext = new Context(activityToInviteTo);
+        model.setContext(newContext);
+
+        return new CommandResult(result, newContext);
```

Comment:
Consider moving this section into the `else` block from between lines 129 - 131, https://github.com/AY1920S1-CS2103T-W11-2/main/blob/33fae232bc864ce38620c6d469125541f5530966/src/main/java/seedu/address/logic/commands/InviteCommand.java#L129-L131 since the UI only needs to be re-rendered if there is at least one person who is successfully invited into the activity.